webapp/pom.xml

```
@@ -45,7 +45,7 @@
     <parent>
         <groupId>fish.payara.monitoring-console</groupId>
         <artifactId>parent</artifactId>
-        <version>1.9</version>
+        <version>1.10-SNAPSHOT</version>
     </parent>
         
     <artifactId>monitoring-console-webapp</artifactId>
```

I'd also add a version tag here
```suggestion
    <artifactId>monitoring-console-webapp</artifactId>
    <version>1.10-SNAPSHOT</version>
```